CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 [Full changelog][unreleased]
 
 - Bumps Ruby version from 3.0.6 to 3.1.2
+- Updates details in IATI xml yaml to reflect switch from BEIS to DSIT
```

Comment:
We'd usually write the changelog and commit messages in an imperative rather than descriptive form (bumps -> bump, updates -> update), but it's not a big deal!

Reply:
Thanks! I can absolutely update that